packages/ripple-binary-codec/test/st-number.test.ts

```
@@ -0,0 +1,114 @@
+import { BinaryParser } from '../src/binary'
+import { coreTypes } from '../src/types'
+
+const { Number } = coreTypes
```

Biome (1.9.4)

[error] Do not shadow the global "Number" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)